packages/data/lib/database/app_database.g.dart

```diff
@@ -0,0 +1,275 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
shouldn't .g.dart files be in git ignore?

Reply:
I made it because file is not generated from root and to generate g file I have to go to package/data and entering long statement It takes me a lot of time.